Sources/XcbeautifyLib/Renderers/OutputRendering.swift

```diff
@@ -565,4 +565,80 @@ extension OutputRendering {
     func formatTestingStarted(group: TestingStartedCaptureGroup) -> String {
         colored ? group.wholeMessage.s.Bold.f.Cyan : group.wholeMessage
     }
+
+    func formatSwiftTestingRunStarted(group: SwiftTestingRunStartedCaptureGroup) -> String {
+        colored ? group.message.s.Bold : group.message
+    }
+
+    func formatSwiftTestingRunCompletion(group: SwiftTestingRunCompletionCaptureGroup) -> String {
+        return "Test run with \(group.numberOfTests) tests passed after \(group.totalTime) seconds"
+    }
+
+    func formatSwiftTestingRunFailed(group: SwiftTestingRunFailedCaptureGroup) -> String {
+        return "Test run with \(group.numberOfTests) tests failed after \(group.totalTime) seconds with \(group.numberOfIssues) issue(s)"
+    }
```

Comment:
Can you add an implementation for the GitHub Actions and TeamCity renderers? I think those one should annotate an error for each.